Source/Core/Core/PowerPC/Jit64IL/IR_X86.cpp

```diff
@@ -420,6 +433,7 @@ static void regNormalRegClear(RegInfo& RI, InstLoc I)
 		regClearInst(RI, getOp2(I));
 }
 
+// Clear any floating point registers which end their lifetime at I
```

Comment:
More precisely, this is supposed to be used for instructions which have no integer operands.

Looking over this code now, I'm not particularly happy with how register allocation is done in general... the code is rather messy, and it's probably not the right tradeoff in terms of performance at JIT time vs. runtime.